test/sql/testing_sql_util.cpp

```diff
@@ -248,7 +249,9 @@ void TestingSQLUtil::ExecuteSQLQueryAndCheckResult(
   // Execute query
   TestingSQLUtil::ExecuteSQLQuery(std::move(query), result, tuple_descriptor,
                                   rows_changed, error_message);
-  unsigned int rows = result.size() / tuple_descriptor.size();
+  unsigned int rows = tuple_descriptor.size() == 0
```

Comment:
Should we add a `PELOTON_ASSERT` instead? I mean, ideally we should only have an empty`result` if the `tuple_descriptor.size()` is 0, right? Or am I missing a corner case?